language/moe/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
The name of the folder and benchmark should be mixtral-8x7b instead of moe (which is too general)
Also we are using the instruct (https://huggingface.co/mistralai/Mixtral-8x7B-Instruct-v0.1) model, we should make it explicit in the readme.